doc/source/ray-core/ray-dashboard.rst

```diff
@@ -6,74 +6,280 @@ Ray provides a web-based dashboard for monitoring and debugging Ray applications
+- **Autoscaler**: Events related to :ref:`Ray autoscaler <cluster-autoscaler>`
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard/event.png
+    :align: center
 
 .. _dash-jobs-view:
 
 Jobs View
-~~~~~~~~~
+---------
 
 The Jobs view lets you monitor the different jobs that ran on your Ray cluster.
-A job is a ray workload that was initiated by an entry point.
-Typically, jobs are initiated via directly calling `ray.init` or a Ray library from a python script or by using the :ref:`job submission api <jobs-overview>`.
 
+A job is a ray workload that uses Ray APIs (e.g., ``ray.init``). It can be submitted directly (e.g., by executing a Python script within a head node) or via :ref:`Ray job API <jobs-quickstart>`.
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard-v2/jobs.png
+    :align: center
+
+The job page displays a list of active, finished, and failed jobs, and clicking on an ID allows users to view detailed information about that job. 
+For more information on Ray jobs, see the Ray Job Overview section.
+
+SANG-TODO Image of the job id with a red circle.
+
+Job Profiling
+~~~~~~~~~~~~~
+
+You can profile Ray jobs by clicking on the “Stack Trace” or “CPU Flame Graph” actions. See the :ref:`Dashboard Profiling <dashboard-profiling>` for more details.
+
+SANG-TODO Image of the profilng buttons with a red circle.
+
+.. _dash-workflow-job-progress:
+
+Advanced Task/Actor Breakdown
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The job page allows you to see tasks and actors broken down by their states. 
+Tasks and actors are grouped and nested by default. You can see the nested entries by clicking the expand button.
+
+Tasks/actors are grouped and nested by the following criteria.
+
+- All tasks and actors are grouped together, and you can view individual entries by expanding the corresponding row.
+- Tasks are grouped by their ``name`` attribute (e.g., ``task.options(name="<name_here>").remote()``).
+- Child tasks (nested tasks) are nested under their parent task's row.
+- Actors are grouped by their class name.
+- Child actors (actors created within an actor) are nested under their parent actor's row.
+- Actor tasks (remote methods within an actor) are nested under the actor for the corresponding actor method.
+
+TODO-SANG Add images
+
+.. note:: 
+
+  Ray dashboard can only display/retrieves up to 10K tasks at a time. If there are more than 10K tasks from your job,
+  they are unaccounted. The number of unaccounted tasks are available from the task breakdown.
+
+Task Timeline
+~~~~~~~~~~~~~
+
+The :ref:`timeline API <ray-core-timeline>` is available from the dashboard. 
+You can download the chrome tracing file by clicking the download button. You can either go to ``chrome://tracing`` or the `Perfetto UI <https://ui.perfetto.dev/>`_ and drop the downloaded chrome tracing file to see the timeline visualization of Ray tasks and actors.
```

Comment:
You can use tools like ``chrome://tracing`` or the `Perfetto UI <https://ui.perfetto.dev/>` and ...